src/query/all_query.rs

```diff
@@ -58,6 +58,15 @@ impl DocSet for AllScorer {
         self.doc
     }
 
+    fn seek(&mut self, target: DocId) -> DocId {
+        debug_assert!(target >= self.doc);
+        self.doc = target;
+        if self.doc >= self.max_doc {
+            self.doc = TERMINATED;
+        }
+        self.doc
```

Comment:
```suggestion
        self.doc = if target_doc >= self.max_doc {
            TERMINATED
        } else {
            target
        };
        self.doc
```

Reply:
I think slightly less nested variant is a little bit more readable